src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ParameterBinder.cs

```diff
@@ -205,6 +205,8 @@ public class ParameterBinder
 
             if (parameter.BindingInfo?.RequestPredicate?.Invoke(actionContext) == false)
```

Comment:
Move the `Logger.AttemptingToBindParameterOrProperty` call above this line. Otherwise, the looking says basically "I gave up" without first logging about trying anything.